schemas/general/schema.json

```diff
@@ -65,6 +65,69 @@
         }
       }
     },
+    {
+      "label": "Pool Level Incentive Allocations",
+      "tableName": "pool_level_incentive_allocations",
+      "aggregation": "general_pools",
+      "uniqueKey": [
+        "pool_address"
```

Comment:
We don't really use this right now, but to be proper the unique key should be pool_address and a timestamp field. You could add `block_date` on the end date of the period and use that.